src/api/app/views/webui2/shared/_editor.html.haml

```diff
@@ -7,21 +7,18 @@
 %div{ class: "card #{'editable' unless style[:read_only]}" }
   - unless style[:read_only]
     .sticky-top.bg-light{ id: "top_#{uid}" }
-      .card-header.d-flex.justify-content-between
-        .input-group.input-group-sm
-          %button.btn.btn-danger.btn-sm.save{ id: "save_#{uid}", disabled: true }
-            %i.fa.fa-save
-            Save
-          %input.form-control.form-control-sm{ id: "comment_#{uid}", type: 'text', size: '40', disabled: true }
-          .btn-group.pl-2
-            %button.btn.btn-secondary.btn-sm{ id: "undo_#{uid}", onclick: "editors[#{uid}].Undo(this);", disabled: true }
-              %i.fa.fa-undo
-              Undo
-            %button.btn.btn-secondary.btn-sm{ id: "redo_#{uid}", onclick: "editors[#{uid}].Redo(this);", disabled: true }
-              %i.fa.fa-redo
-              Redo
-        %button.btn.btn-warning.btn-sm{ data: { 'target': "#settings#{uid}", 'toggle': 'modal' } }
-          %i.fa.fa-cog
+      .card-header
+        .d-flex.justify-content-between
+          .input-group.input-group-sm
```

Comment:
Theses classes are needed? :thinking:. We don't have any input here

Reply:
They are not. I'll remove them!